hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java

```diff
@@ -408,6 +421,15 @@ public KeyInfo getProtobuf() {
     if (encInfo != null) {
       kb.setFileEncryptionInfo(OMPBHelper.convert(encInfo));
     }
+    if (ignorePipeline) {
+      kb.addAllKeyLocationList(keyLocationVersions.stream()
+          .map(OmKeyLocationInfoGroup::getCompactProtobuf)
+          .collect(Collectors.toList()));
+    } else {
+      kb.addAllKeyLocationList(keyLocationVersions.stream()
+          .map(OmKeyLocationInfoGroup::getProtobuf)
+          .collect(Collectors.toList()));
+    }
```

Comment:
OmKeyInfo was changed in #1034 to avoid streams.  `keyLocationVersions` is now converted in a loop:

https://github.com/apache/hadoop-ozone/blob/2af6198686d81daa3ad0513f723118637d2945cf/hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java#L388-L391

That's where normal/compact protobuf conversion should happen depending on `ignorePipeline` parameter.